session.go

```diff
@@ -641,7 +641,7 @@ func (s *session) handleTCPPackage() error {
 		for {
 			// for clause for the network timeout condition check
 			// s.conn.SetReadTimeout(time.Now().Add(s.rTimeout))
-			bufLen, err = conn.read(buf)
+			bufLen, err = conn.read(buf[:maxReadBufLen])
```

Comment:
I think `buf[:maxReadBufLen]` is the same as `buf`.

Reply:
I agree with u. just rollback the codes.